workbench/tests/renderer/app.test.js

```diff
@@ -297,6 +298,7 @@ describe('Display recently executed InVEST jobs on Home tab', () => {
         results_suffix: 'suffix',
       },
       status: 'error',
+      type: 'core',
```

Comment:
For backwards compatibility we want to make sure that a job with no `type` property is treated like a "core" model. That seems to be true since the typical check is for `(job.type === 'plugin')` which evaluates to `(undefined === 'plugin')`. We could test that implicitly here by omitting the `type: 'core'` property.

Reply:
good idea!